EnglishTranslation-master/ETC.tsv

```diff
@@ -6335,67 +6335,67 @@ ETC_20150317_006335	Salavija Hill
 ETC_20150317_006336	Abandoned Camp
 ETC_20150317_006337	Hollow Tree Forest
 ETC_20150317_006338	Field of Reeds
-ETC_20150317_006339	Ishiblysh Square
+ETC_20150317_006339	Isblyskes Square
 ETC_20150317_006340	Ascetic Sanctuary
 ETC_20150317_006341	Nevasingas Highway
-ETC_20150317_006342	Kabeti Hill
+ETC_20150317_006342	Kabantis Hill
 ETC_20150317_006343	Nomads Camp
-ETC_20150317_006344	Zakole Shore
+ETC_20150317_006344	Pusmenulio Shore
 ETC_20150317_006345	Pilgrim's Tomb
-ETC_20150317_006346	Pasukti Path
-ETC_20150317_006347	Noreti Fountain
+ETC_20150317_006346	Plunksnos Path
```

Comment:
Wiki says it's "turn", so "Posukis".
"Plunksnos" - "of the feather". "Feather" = "Plunksna".